source/sass/patterns/molecules/figure.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../variables/breakpoint";
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.figure {
+  @include block-spacing($end: $baselinegrid-space-medium);
+}
+
+.figure__image {
+  @include block-spacing($start: $baselinegrid-space-extra_small, $end: $baselinegrid-space-extra_small);
+  @include margin(12px, inline);
+  border: 1px solid $color-text-dividers;
+  display: block;
+
+}
+
+.figure__heading {
+  @include caption-typography();
+}
+
+.figure__caption {
+  @include caption-typography();
+  @include margin(12px, inline);
+}
+
+@media only screen and (min-width: #{($breakpoint-site-x_small)}) {
```

Comment:
There's a mixin for this: https://github.com/libero/pattern-library/blob/17b8c3eb95ba231aa684af76ee84549a775db82a/source/sass/patterns/organisms/item-tags.scss#L13

In this case, `$breakpoint-site-x_small` is actually useless as it is the minimum screen width. eLife has a breakpoint at a bigger size.